------------------------------------------------------------
revno: 14155
revision-id: squid3@treenet.co.nz-20170504061416-ks61dfut8wyml2qu
parent: squid3@treenet.co.nz-20170402121452-ox6d8ttzlmbov3xm
fixes bug: http://bugs.squid-cache.org/show_bug.cgi?id=4682
author: Christos Tsantilas <chtsanti@users.sourceforge.net>
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Thu 2017-05-04 18:14:16 +1200
message:
  Bug 4682: Fix ssl_bump "bump" action documentation
  
  Fixes squid documentation to correctly describe the squid behavior  when the
  "bump" action is selected on step SslBump1. In this case squid selects
  the client-first bumping mode.
  
  This is a Measurement Factory project
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170504061416-ks61dfut8wyml2qu
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: f3b4861a085e069948da25398782237609037c5f
# timestamp: 2017-05-04 06:16:54 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170402121452-\
#   ox6d8ttzlmbov3xm
# 
# Begin patch
=== modified file 'src/cf.data.pre'
--- src/cf.data.pre	2017-03-31 23:38:31 +0000
+++ src/cf.data.pre	2017-05-04 06:14:16 +0000
@@ -2669,8 +2669,11 @@
 		This is the default action.
 
 	    bump
-		Establish a secure connection with the server and, using a
-		mimicked server certificate, with the client.
+		When used on step SslBump1, establishes a secure connection
+		with the client first, then connect to the server.
+		When used on step SslBump2 or SslBump3, establishes a secure
+		connection with the server and, using a mimicked server
+		certificate, with the client.
 
 	    peek
 		Receive client (step SslBump1) or server (step SslBump2)

